configuration/src/main/java/io/camunda/configuration/beans/SearchEngineConnectProperties.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.configuration.beans;
+
+public class SearchEngineConnectProperties extends LegacySearchEngineConnectProperties {}
```

Comment:
@buccarel This should not extend the legacy, instead should extend "ConnectConfiguration" to prevent two beans of same type.